torch/autograd/functional.py

```diff
@@ -342,18 +341,37 @@ def jvp(func, inputs, v=None, create_graph=False, strict=False):
                                "the user-provided function is a single Tensor "
                                "with a single element.")
 
-    outputs = func(*inputs)
-    is_outputs_tuple, outputs = _as_tuple(outputs, "outputs of the user-provided function", "jvp")
-    _check_requires_grad(outputs, "outputs", strict=strict)
-    # The backward is linear so the value of grad_outputs is not important as
-    # it won't appear in the double backward graph. We only need to ensure that
-    # it does not contain inf or nan.
-    grad_outputs = tuple(torch.zeros_like(out, requires_grad=True) for out in outputs)
+    if fw_mode:
+        if v is None:
+            v = (torch.ones_like(inputs[0]),)
```

Comment:
nit: assert (or write a comment) that `inputs` is expected to have exactly 1 element. It took me a while to figure out that invariant